### PR TITLE
Bugfixes for 3.0.3

### PR DESCRIPTION
**Pull request by bgreiner**
These are a number of bugfixes that were implemented in responding to user requests. They should be part of a minor update 3.04.

### CODE REVIEW

File: admin/admin_edit.php [hunk 1]
```diff
@@ -84,6 +84,7 @@
             } else unset($_REQUEST['password']);
 
             if (!$admin_id) $admin_id=time();
+            foreach (array('fname','lname') as $k) $_REQUEST[$k]=trim($_REQUEST[$k]);
```
Comment (kabalin) [inline admin/admin_edit.php]: Ben, three things here:

1. to what extent we follow the https://github.com/orsee/orsee/wiki/Coding-style, if it is strict then this line should have brackets to make it easier to follow.
2. I am not sure if array values trimming needs to be done here in one-by-one fashion, it should be applied to any array passed as the first parameter within orsee_db_save_array function, if that is possible (unless there is a case when text value needs to contain spaces at either end).
3. "Username ist lost" <- typo in commit message.

**Comment by bgreiner** [comment]
Honestly, I have no idea how to do this with "git cherry-pick". I tried to read up on this, but I don't get it. And I didn't want to violate our policy of only directly merging into develop, so that the network graph stays clean.

File: admin/admin_edit.php (hunk 1 above)
Comment (bgreiner) [inline admin/admin_edit.php]: I fixed 1. and 3. Re 2., I think there was some reason for only rimming fname and lname, though I do not remember it ... I would just leave as is for the moment, to not run into any trouble.

**Comment by kabalin** [comment]
OK, let's merge this one without modifications for now. "git cherry-pick" just brings someone's commit directly to your branch, on the newrotk graph it will not have reference to its original branch at all, it will look as single commit in line with others, but just with a different author.

To do it, you need to add that person's repo as "remote" (git remote add dnacombo https://....) (https://help.github.com/articles/adding-a-remote/), then "git fetch dnacombo", then find the hash of commit you need to cherry pick (git log dnacombo/bugfixes_for_3.0.3) and "git cherry-pick <HASH>", this will bring a single commit to your current branch. if you are not happy with commit message, you may do "git commit --amend" and change it, but author will be preserved. When you finished, just "git remote rm dnacombo" (https://help.github.com/articles/removing-a-remote/)

**Comment by kabalin** [comment]
Also, you do not need to a separate commit for addressing comments (unless you want). Just amend code in existing commit and force push the feature branch.

**Comment by bgreiner** [comment]
Yes, I know. But changing old commits and force-pushing is 1) more work than just adding a new commit, and 2) may change the commit id, no? The disadvantage is that original commits cannot be referred to as a "fix" of a particular bug, since a later commit amends that fix ...

**Pull request merged**